core/codenvy-hosted-api-workspace/src/main/java/com/codenvy/api/workspace/server/filters/WorkspacePermissionsFilter.java

```diff
@@ -79,7 +79,6 @@ public void filter(GenericResourceMethod genericResourceMethod, Object[] argumen
                 workspaceId = ((String)arguments[0]);
                 action = DELETE;
                 break;
-            case "recoverWorkspace":
```

Comment:
@sleshchenko is there any common mechanism which checks whether such methods exist?
I think this code must be covered by tests, as we usually have troubles in places like this after refactorings. 
@vinokurig consider adding such tests